spec/lib/scriptable_spec.rb

```diff
@@ -343,6 +343,46 @@
           expect(Topic.last.allowed_groups).to contain_exactly(group)
         end
       end
+
+      context "when pm target_emails with valid email" do
+        it "sends the pm" do
+          expect {
+            DiscourseAutomation::Scriptable::Utils.send_pm(
+              {
+                title: "Private Message Title",
+                raw: "0123456789" * 25 + "a",
+                target_emails: ["john@doe.com"],
+              },
+            )
+          }.to change { Topic.count }
```

Comment:
Might just want to scope these a bit more so it's clearer what it's testing for `Topic.private_messages.count`

Reply:
Also might want to add a test that checks multiple PMs are made for multiple PM email recipients